flink-connectors/flink-connector-kafka/src/test/java/org/apache/flink/connector/kafka/sink/KafkaRecordSerializationSchemaBuilderTest.java

```diff
@@ -45,15 +46,16 @@
 import static org.assertj.core.api.Assertions.assertThatThrownBy;
 
 /** Tests for {@link KafkaRecordSerializationSchemaBuilder}. */
-public class KafkaRecordSerializationSchemaBuilderTest extends TestLogger {
+@ExtendWith(TestLoggerExtension.class)
+public class KafkaRecordSerializationSchemaBuilderTest {
 
     private static final String DEFAULT_TOPIC = "test";
 
     private static Map<String, ?> configurableConfiguration;
     private static Map<String, ?> configuration;
     private static boolean isKeySerializer;
 
-    @Before
+    @BeforeEach
     public void setUp() {
```

Comment:
```suggestion
    void setUp() {
```
could be package private